tests/Benchmarks/System.Binary.Base64/Stiching.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using BenchmarkDotNet.Attributes;
```

Comment:
nit: add a new line between the license header and the using directives.

For example: https://github.com/dotnet/corefxlab/blob/67620c5a10103a72c5125cc85ac123e56943ed60/src/System.Binary.Base64/Base64Encoder.cs#L4